client/src/store/new-request-module.ts

```diff
@@ -2571,13 +2563,16 @@ export class NewRequestModule extends VuexModule {
       if (this.quickSearch) {
         this.mutateDisplayedComponent('QuickSearchResults')
       }
-    } catch (error) {
-      // send error to sentry and move on to detailed search (silently skips error for user)
-      console.error('getQuickSearch() =', error) // eslint-disable-line no-console
+    } catch (err) {
+      const msg = await handleApiError(err, 'Could not get quick search')
+      // send error to sentry and move on to detailed search
+      // (do not show error to user)
+      console.error('getQuickSearch() =', msg) // eslint-disable-line no-console
```

Comment:
This should probably be reworked at some point, as I'm not sure we want to capture (ie, Sentry) all quick search errors. But maybe some of them? I dunno.